app/src/utils/apiRequest.js

```diff
@@ -0,0 +1,33 @@
+export const request = (url, method, bodyParams) => {
+    return fetch(url, {
+        method,
+        headers: {
+            'Content-Type': 'application/json',
+        },
+        body: bodyParams ? JSON.stringify(bodyParams) : undefined,
+    });
+};
+
+function withToken(httpConfig) {
+    try {
+        const user = JSON.parse(localStorage.getItem('user'));
+        httpConfig.headers['Authorization'] = `jwt_token ${user}`;
+    } catch (err) {
+        throw (err);
```

Comment:
That is a little bit strange to catch an error and throw the news one, don't you think so? You can just remove try/catch and you will get the same result.